cypress/page/menu-content.page.ts

```diff
@@ -0,0 +1,19 @@
+class MenuContentPage {
+    private tShirtMenu: string;
+    private menuContentPageURL: string
+
+    constructor() {
+        this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
```

Comment:
In general, is not a good approach to use the `nth-child`  function because we couple the locator to the position of a specific element, and try to use a the title attribute to locate the t-shirt menu.